### ISSUE TITLE
Profile GitLab Link

### ISSUE THREAD

**Issue by jeremylt** [freeCodeCamp/freeCodeCamp#55194]
### Is your feature request related to a problem? Please describe.

For the learn platform, we have a spot to link to GitHub, Linkedin, and Twitter. Why not add GitLab to that as well?

### Describe the solution you'd like

Ability to set link to GitLab profile

### Describe alternatives you've considered

N/A

**Comment by shachargiladi** [comment]
Hey, can I work on this issue?

**Comment by Abhaysoft-inc** [comment]
I tried to add this feature but don't know why the data is not getting saved, I was able to create the frontend but I was not able to upadte the gitlab link. I think i don't have knowledge about prisma. I will figure it it soon.

![Screenshot (1)](https://github.com/freeCodeCamp/freeCodeCamp/assets/57512017/9cb0398f-c818-4476-9335-b3eaf513d589)

**Comment by Abhaysoft-inc** [comment]
can anyone help me implementing this?

**Comment by ilenia-magoni** [comment]
This issue is not open for contribution. Please contribute to issues with the help wanted label.

**Comment by Abhaysoft-inc** [comment]
Sorry and thanks for telling

**Comment by naomi-lgbt** [comment]
I'm going to loop @ahmaxed in on this, as he's been most engaged with the redesign I think?

Either way, we'll probably want to fold this in to https://github.com/freeCodeCamp/freeCodeCamp/issues/49631.

**Comment by ahmaxed** [comment]
Thanks for opening the issue.

**Comment by ahmaxed** [comment]
Thanks for the feature request. We have received a few platform link addition requests over the years, but decided to stick with the most popular platforms that are relevant to our users for simplicity.

If we are going to add additional platforms, we would need a comprehensive solution that allows users to choose from a list of platforms.

Here are the following tasks that need to be completed before the new feature is added:

1. Research similar solutions
2. Alter the settings/profile UI to account for the new functionality
3. client/api validations
4. update api endpoint and user schema

Since a more comprehensive platform link solution would be out of scope for this issue, I am closing this issue accordingly.
